docs/tas5825m.md

```diff
@@ -0,0 +1,80 @@
+# TAS5825M
+
+The TAS5825M is an audio amp for speakers.
+
+## Porting to coreboot
+
+There is a TAS5825M driver in the System76 fork of coreboot, but each board
+must add the code to initialize the device.
+
+Items required:
+
+- Saleae logic analyzer
+- Saleae Logic 2
+- Micro SMD grabber test clips
+
+Be sure to unplug the battery before starting.
+
+### Attaching test clips
+
+It may not be feasible to probe the TAS5825M chip itself. Instead look
+for the MOSFET that connects to `SCL_5825` and `SDA_5825`, which will
```

Comment:
These lines often have different names. I'd just say "the smart amp's I2C clock and data lines"